steelscript/appresponse/core/reports.py

```diff
@@ -31,7 +31,6 @@ class SourceProxy(object):
 
     def __init__(self, packets_obj=None, name=None, path=None):
         """Initialize a data source for reports to run against.
-
         :param packets_obj: Clip or File or Job object
```

Comment:
A lot of blank lines are removed - not sure if intentional?  They help visually distinguish paragraphs in the docstrings though, so they should be re-inserted.